repos/fdbt-netex-output/src/netex-validator/main.py

```diff
@@ -99,13 +130,28 @@ def lambda_handler(event, context):
 
         try:
             logger.info('Starting NeTEx validation...')
-            validate_netex(download_path)
 
-            logger.info('NeTEx valid, uploading to S3...')
-            s3_client.upload_file(download_path, os.getenv('VALIDATED_NETEX_BUCKET'), key)
+            netex_file = open(download_path, 'r+', encoding="ISO-8859-1")
```

Comment:
This can just be r since we're only reading

Reply:
Will change